shelley/chain-and-ledger/executable-spec/test/STSTests.hs

```diff
@@ -0,0 +1,107 @@
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE TypeApplications #-}
+
+module STSTests (stsTests) where
+
+import           Data.Either (isLeft)
+import qualified Data.Map.Strict as Map
+import           Data.Maybe (fromMaybe)
+import           Data.Word (Word64)
+import           Test.Tasty
+import           Test.Tasty.HUnit
+
+import           Cardano.Crypto.KES (deriveVerKeyKES, genKeyKES)
+import           Crypto.Random (drgNewTest, withDRG)
+import           MockTypes
```

Comment:
Don't we want to import everything implicitly?

Reply:
explicitly? are you saying `MockTypes` is the oddball here since I didn't just take what I needed? I'm happy to do whatever!

Reply:
Yes, and also the `Tasty` imports. Maybe we want to be consistent and import everything explicitly, but whatever is more convenient for you.

Reply:
I like the idea explicitly importing everything, thanks!